packages/anchors/package.json

```diff
@@ -9,11 +9,11 @@
     "compile": "tsc -p tsconfig.build.json"
   },
   "dependencies": {
-    "@webb-tools/contracts": "^0.3.0",
-    "@webb-tools/interfaces": "^0.3.0",
+    "@webb-tools/contracts": "^0.3.3-alpha.0",
```

Comment:
I'm not sure what the current steps are for publishing - previously a `yarn publish` would bump these versions and publish with the properly updated dependencies.

Should these versions be changed?